service/src/main/java/org/cbioportal/service/impl/StructuralVariantServiceImpl.java

```diff
@@ -23,170 +23,37 @@
 
 package org.cbioportal.service.impl;
 
-import org.apache.commons.collections4.CollectionUtils;
 import org.cbioportal.model.GeneFilterQuery;
 import org.cbioportal.model.StructuralVariant;
-import org.cbioportal.persistence.MutationRepository;
 import org.cbioportal.persistence.StructuralVariantRepository;
 import org.cbioportal.service.StructuralVariantService;
-import org.cbioportal.service.util.MolecularProfileUtil;
-import org.cbioportal.service.util.MutationMapperUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import java.util.*;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-import org.cbioportal.model.MolecularProfile;
-import org.cbioportal.persistence.MolecularProfileRepository;
+
```

Comment:
blank line not needed. maybe this is to divide our own classes from standard library classes?